xpublish/rest.py

```diff
@@ -131,9 +167,9 @@ def register_plugin(
         Register a plugin with the xpublish system
 
         Args:
```

Comment:
Shouldn't this be converted to a 
```
Parameters
----------------
```

section?

Reply:
> I've just stumbled upon this change, because I wanted to know how you use sphinx-autodoc-typehints in a napoleon project.

friendly ping @abkfenris